crates/oxc_linter/src/snapshots/prefer_spread.snap

```diff
@@ -0,0 +1,659 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: prefer_spread
+---
+  ⚠ eslint-plugin-unicorn(prefer-spread): Prefer the spread operator over Array.from()
+   ╭─[prefer_spread.tsx:1:1]
+ 1 │ const x = Array.from(set);
+   ·           ───────────────
```

Comment:
I don't really get what the error message is talking about, or more precisely I don't get what I need to do to fix the error if I don't understand what a spread operator is.

Reply:
I've updated the error/help. Would adding a `fix` for all the cases also be a solution